sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Fixed async `update_sync_token` to use async/await keywords
 
 ### Other Changes
+- Set the default `api_version` to "1.0".
```

Comment:
since there are new operations added they should be under new features :)

Reply:
Ah, you're right, I missed that.

Reply:
Updated!